### ISSUE TITLE
wallet aliases

### ISSUE THREAD

**Issue by yegor256** [zold-io/zold#279]
Let's make it possible to assign aliases to wallets. Let's create a file `.zoldata/aliases` with two columns: wallet ID and alias. The alias has to match `/^[a-zA-Z0-9\-_]$/`. To assign it:

```
$ zold alias set 17737fee5b825835 wts
```

Here, the wallet `17737fee5b825835` will get alias `wts`. To remove it:

```
$ zold alias remove 17737fee5b825835
```

To show current alias of a wallet:

```
$ zold alias show 17737fee5b825835
wts
```

Or just

```
$ zold alias 17737fee5b825835
There is no alias for the wallet 17737fee5b825835, you can set it with 'zold alias set'
```

If the alias is set, it must be possible to use it everywhere we use wallet ID. For example:

```
$ zold show wts
$ zold pay wts John 15 'Thanks, John!'
$ zold push wts
```

**Comment by 0crat** [comment]
Job [#279](https://github.com/zold-io/zold/issues/279) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by yegor256** [comment]
@0crat assign @botanicus

**Comment by 0crat** [comment]
> @0crat assign @botanicus ([here](https://github.com/zold-io/zold/issues/279#issuecomment-399963656))

@yegor256 The job [#279](https://github.com/zold-io/zold/issues/279) assigned to @botanicus[/z](https://www.0crat.com/u/botanicus), here is [why](https://www.0crat.com/footprint/CAZPZR9FS/1761436859); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
@botanicus[/z](https://www.0crat.com/u/botanicus) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#322](https://github.com/zold-io/zold/issues/322), [#323](https://github.com/zold-io/zold/issues/323) are still not solved.

**Comment by botanicus** [comment]
@yegor256 please close this.

**Comment by yegor256** [comment]
@botanicus thanks!

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @botanicus[/z](https://www.0crat.com/u/botanicus), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#279](https://github.com/zold-io/zold/issues/279) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +38 point(s) just awarded to @botanicus[/z](https://www.0crat.com/u/botanicus)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@yegor256 4 puzzles [#323](https://github.com/zold-io/zold/issues/323), [#377](https://github.com/zold-io/zold/issues/377), [#378](https://github.com/zold-io/zold/issues/378), [#379](https://github.com/zold-io/zold/issues/379) are still not solved; solved: [#322](https://github.com/zold-io/zold/issues/322).
